stix_shifter_modules/elastic_ecs/stix_translation/json/to_stix_map.json

```diff
@@ -196,30 +609,188 @@
     }
   },
   "process": {
-    "executable": {
-      "key": "process.command_line",
-      "object": "proc"
-    },
     "start": {
       "key": "process.created",
-      "object": "proc"
+      "object": "process"
+    },
+    "pid": [
+      {
+        "key": "process.pid",
+        "object": "process",
+        "transformer": "ToInteger"
+      },
+      {
+        "key": "process.parent_ref",
+        "object": "process",
+        "references": "process_ppid"
+      }
+    ],
+    "name": {
+      "key": "process.name",
+      "object": "process"
+    },
+    "code_signature": {
+      "exists": {
+        "key": "x_ecs_process.code_signature_exists",
+        "cybox": false
+      },
+      "subject_name": {
+        "key": "x_ecs_code_process.signature_subject_name",
+        "cybox": false
+      }
     },
-    "pid": {
+    "ppid": {
       "key": "process.pid",
-      "object": "proc",
+      "object": "process_ppid",
       "transformer": "ToInteger"
     },
-    "name": [
-      {
-        "key": "file.name",
-        "object": "file"
+    "pe": {
+      "company": {
+        "key": "x_ecs_process.pe_company",
+        "cybox": false
       },
-      {
-        "key": "process.image_ref",
-        "object": "proc",
-        "references": "file"
+      "description": {
+        "key": "x_ecs_process.pe_description",
+        "cybox": false
+      },
+      "file_version": {
+        "key": "x_ecs_process.pe_file_version",
+        "cybox": false
+      },
+      "original_file_name": {
+        "key": "x_ecs_process.pe_original_file_name",
+        "cybox": false
+      },
+      "product": {
+        "key": "x_ecs_process.pe_product",
+        "cybox": false
       }
-    ]
+    },
+    "args": {
+      "key": "x_ecs_process.args",
+      "cybox": false
+    },
+    "args_count": {
+      "key": "x_ecs_process.args_count",
+      "cybox": false
+    },
+    "command_line": {
+      "key": "process.command_line",
+      "object": "process"
+    },
+    "executable": {
+      "key": "x_ecs_process.executable",
+      "cybox": false
+    },
+    "entity_id": {
+      "key": "x_ecs_process.entity_id",
+      "cybox": false
+    },
+    "exit_code": {
+      "key": "x_ecs_process.exit_code",
+      "cybox": false
+    },
+    "parent": {
+      "args": {
+        "key": "x_ecs_process.parent.args",
+        "cybox": false
+      },
+      "args_count": {
+        "key": "x_ecs_process.parent.args",
+        "cybox": false
+      },
+      "command_line": {},
```

Comment:
The command_line entry is empty. Will want to add `x_ecs_process.parent_command_line`

Reply:
my bad. added